tools/mark-blocks/README.md

```diff
@@ -17,6 +17,25 @@ This program creates or removes markers for blocks.
 
 Each supported object storage service also has an additional set of flags (see examples in [Running](##Running)).
 
+## Input formats
+
+For convenience, this tool supports two input formats controlled by the `--tenant` flag.
+
+If `--tenant` is empty (the default), then a tenant must be specified for each block provided in either `--blocks` or `--blocks-file`. For example, an input file could be of the form:
```

Comment:
Should we add an example for when multi-tenancy is disabled and we use the fallback tenant?

Reply:
Actually, I guess I don't know how that would even work offhand. `--tenant=anonymous` ?

Reply:
Yeah, we don't assume `anonymous` here and never did for this tool. It used to fail if `--tenant` wasn't specified.

You'd either have to specify `--tenant=anonymous` or denote it in the input like `--blocks="anonymous/<blockID1>,anonymous/<blockID2>`, which aligns with the other examples.